packages/slice-machine/src/features/settings/git/useNestedGroupExperiment.ts

```diff
@@ -0,0 +1,8 @@
+import { useExperimentVariant } from "@/hooks/useExperimentVariant";
+
+type UseNestedGroupExperimentReturnType = { eligible: boolean };
+
+export function useNestedGroupExperiment(): UseNestedGroupExperimentReturnType {
```

Comment:
⚠️ #issue
 
You put it under git folder that is specific to all git features. If on your side it's not related to a specific feature that I think it's not, you can put the hook directly in hooks folder.

Reply:
Good catch, I moved it.